spec/models/character_spec.rb

```diff
@@ -0,0 +1,18 @@
+RSpec.describe Character do
+  Stat.create!(name: "Speed")
+
+  let(:character) { Character.create!(name: "Professor Longfellow") }
+  let(:character_stat) { CharacterStat.create!(character_id: character.id,
+                                               stat_id: 1,
+                                               level: 2,
+                                               value: 3)
+}
```

Comment:
`end` at 9, 0 is not aligned with `let(:character_stat) { CharacterStat.create!(character_id: character.id,` at 5, 2